src/common/regions/utils.ts

```diff
@@ -0,0 +1,33 @@
+import { concat, partition } from 'lodash';
+import regions from './region_db';
+import { getStateFips } from './regions_data';
+import { County, State, Region, MetroArea } from './types';
+
+export function belongsToState(county: County, stateFips: string | null) {
+  return county.state.fipsCode === stateFips;
+}
+
+export function getAutocompleteLocations(region?: Region) {
```

Comment:
nit: Would prefer this to be `getAutocompleteRegions` as we have the `Location` and `Region` object. Also, will type inference work properly here? 

Also, maybe a name like `getAllAutocompleteRegions` or `getAllAutocompleteRegionsSorted` might be more clear? Though I don't love those names usggestions.  I think a comment here might be useful to indicate what the `region` argument does.  it's not clear from the function signature that region changes the sort order of the list.